src/App.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import logo from './logo.svg';
 import { Counter } from './features/counter/Counter';
 import './App.css';
+import FieldInput from './features/input/FieldInput';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <Counter />
+        <FieldInput/>
```

Comment:
Space before the closing of tag should be in both cases, or in none of them.
Optionally, you can configure eslint to do it automatically - https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-space-before-closing.md